src/__tests__/posthog-persistence.test.ts

```diff
@@ -115,12 +116,12 @@ describe('persistence', () => {
             const lib = new PostHogPersistence(makePostHogConfig('bla', 'cookie'))
             lib.register_once({ distinct_id: 'testy', test_prop: 'test_value' }, undefined, undefined)
             expect(document.cookie).toEqual(
-                '; ph__posthog=%7B%22distinct_id%22%3A%22testy%22%2C%22test_prop%22%3A%22test_value%22%7D'
+                'ph__posthog=%7B%22distinct_id%22%3A%22testy%22%2C%22test_prop%22%3A%22test_value%22%7D'
```

Comment:
checked this locally with yalc and cookies seem correct 
i guess this was a test artefact 🤷